02_bubble/saurabhvyas9/main_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := strconv.Quote("[1 2 3 5][1 2 3 5]")
+	actual := strconv.Quote(buf.String())
+	if expected != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %s\nactual: %s", expected, actual)
+	}
+}
+
+var testCases = map[string]struct {
+	input []int
+	want  []int
+}{
+	"TC1": {input: []int{3, 1, 2, 5}, want: []int{1, 2, 3, 5}},
```

Comment:
nice! love the table driven test.
Maybe choose more meaningful names: e.g. `Simple`, `Sorted`, `Duplicates`...
Also add single element and empty slices please.

Reply:
Done